src/filterBar/FilterBar.unit.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+
+import { fireEvent, render, screen } from '@testing-library/react'
+
+import { BusIcon } from '../icon/busIcon'
+import { CarpoolIcon } from '../icon/carpoolIcon'
+import { FilterBar, FilterBarProps } from '.'
+
+const defaultProps: FilterBarProps = {
+  ctaText: 'See rides',
+  supplyInfo: [
+    {
+      liquidity: 98,
+      icon: CarpoolIcon,
+      ariaLabel: 'Carpooling',
+    },
+    {
+      liquidity: 12,
+      icon: BusIcon,
+      isDisabled: true,
+      ariaLabel: 'Bus',
+    },
+  ],
+  onClick: () => {},
+}
+
+function createProps(props: Partial<FilterBarProps> = {}): FilterBarProps {
+  return { ...defaultProps, ...props }
+}
+
+describe('FilterBar', () => {
+  it('should render icons & liquidity and cta', () => {
```

Comment:
This test doesn't check the presence of icons but if you want to you can use `getByRole('image', { 
name: 'Bus' })` or `getByAltText('Bus')`.

Reply:
I can't it's not an `img` tag but an `svg` with `aria-hidden=true`

Reply:
Added a iconTitle prop and target it using `getByTitle`